File: Typecast/TCProgram.m
//
//  TCProgram.m
//  Typecast
//
//  Created by David Schweinsberg on 9/12/12.
//  Copyright (c) 2012 David Schweinsberg. All rights reserved.
//

#import "TCProgram.h"
#import "Type_Designer-Swift.h"

@implementation TCProgram

- (void)readInstructionsWithDataInput:(TCDataInput *)dataInput length:(NSUInteger)length
{
  [dataInput readWithLength:length];
//    _instructions = [dataInput readDataWithLength:length];
}

@end

File: Typecast/TCProgram.swift
//
//  TCProgram.swift
//  Type Designer
//
//  Created by David Schweinsberg on 8/3/17.
//  Copyright © 2017 David Schweinsberg. All rights reserved.
//

import Foundation
import IOUtils

class TCProgram: NSObject {
  var instructions = [UInt8]()

  func readInstructions(dataInput: TCDataInput, count: Int) {
    instructions = dataInput.read(length: count)
  }
}
